activejob/test/cases/async_job_test.rb

```diff
@@ -0,0 +1,67 @@
+require 'helper'
+require 'jobs/kwargs_job'
+
+class AsyncJobTest < ActiveSupport::TestCase
+
+  def job_queue_spy
+    ActiveJob::AsyncJob.const_get(:QUEUES)
```

Comment:
Making the constant public (with a `# :nodoc:`) would feel more apt than disregarding what you've marked it as :smile: